star.wdl

```diff
@@ -95,6 +95,10 @@ task Star {
         String outFileNamePrefix
         String outSAMtype = "BAM SortedByCoordinate"
         String readFilesCommand = "zcat"
+        Int outFilterScoreMin = 0
+        Float outFilterScoreMinOverLread = 0.66
+        Int outFilterMatchNmin = 0
+        Float outFilterMatchNminOverLread = 0.66
```

Comment:
Isn't it better to use these as `Int?` and `Float?`. The defaults set by STAR might change with newer versions and then we are stuck with these old defaults.